From duncan.sands@math.u-psud.fr Fri Apr 28 09:59:04 2006
From: Duncan Sands <duncan.sands@math.u-psud.fr>
To: Greg KH <greg@kroah.com>
Subject: USBATM: remove no-longer needed #include
Cc: linux-usb-devel@lists.sourceforge.net
Content-Disposition: inline
Date: Fri, 28 Apr 2006 18:58:57 +0200
Message-Id: <200604281858.57636.duncan.sands@math.u-psud.fr>

We #include <linux/netdevice.h> only because <linux/etherdevice.h>
needed it, but didn't #include it itself.  But that's been fixed now.

Signed-off-by: Duncan Sands <baldrick@free.fr>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/atm/xusbatm.c |    1 -
 1 file changed, 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/atm/xusbatm.c
+++ gregkh-2.6/drivers/usb/atm/xusbatm.c
@@ -20,7 +20,6 @@
  ******************************************************************************/
 
 #include <linux/module.h>
-#include <linux/netdevice.h>		/* FIXME: required by linux/etherdevice.h */
 #include <linux/etherdevice.h>		/* for random_ether_addr() */
 
 #include "usbatm.h"
